<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RingFileSetDialog</class>
 <widget class="QDialog" name="RingFileSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="CommonPushButton" name="pushButtonRingFileSetSave">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>380</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QFrame" name="framePanel">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>12</y>
     <width>700</width>
     <height>570</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonRingFileSetCancel">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>480</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewRingFile">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>92</y>
     <width>660</width>
     <height>480</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>22</y>
     <width>510</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>铃声</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonRingFileSetSave</tabstop>
  <tabstop>pushButtonRingFileSetCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
